L1Trigger/DTTriggerPhase2/plugins/DTTrigPhase2Prod.cc

```diff
@@ -866,4 +878,89 @@ void DTTrigPhase2Prod::processDigi(std::queue<std::pair<DTLayerId, DTDigi>>& inQ
   vec.push_back(&newQueue);
 }
 
+void DTTrigPhase2Prod::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // dtTriggerPhase2PrimitiveDigis
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("digiTag", edm::InputTag("CalibratedDigis"));
+  desc.add<int>("timeTolerance", 999999);
+  desc.addUntracked<std::string>("geometry_tag", "");
+  desc.addUntracked<int>("trigger_with_sl", 4);
```

Comment:
Am I wrong, or several of these parameters are never used in the code? If so, could you please clean them up?